oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongodProcessFactory.java

```diff
@@ -145,7 +147,8 @@ private void initRS(String rs, int[] ports) {
         }
         Document config = new Document("_id", rs);
         config.append("members", members);
-        try (MongoClient c = new MongoClient(localhost(), ports[0])) {
+        ConnectionString mongoURI = new ConnectionString(localhost());
+        try (MongoClient c = MongoClients.create(localhost())) {
```

Comment:
This does not look right to me. `mongoURI` is never used and IIUC the MongoClient created here will try to connect to a MongoDB instance on the default port, which isn't necessarily the case.